src/avl/CMakeLists.txt

```diff
@@ -12,13 +12,13 @@ set(target ${LIBRARY})
 file(GLOB hrcs "AVL.h")
 file(GLOB srcs "AVL.cpp")
```

Comment:
```suggestion
file(GLOB srcs "AVL.cpp" "AVLNode.cpp")
```
or for all files with extension 

```suggestion
file(GLOB srcs "*.cpp")
```

same below

Reply:
done